steve_configuration.bat

```diff
@@ -0,0 +1,27 @@
+@echo off
+echo -- Steve --
+echo Creating conda environment
+
+conda create -n steve steve
```

Comment:
Steve was already installed somewhere.
Creating an env with a soo common name seem quite prone to problems.

Reply:
For a multiplatform and more standard solution, I'd suggest checking either [conda-execute](https://github.com/conda-tools/conda-execute) or [conda-wrappers](https://github.com/gqmelo/exec-wrappers).

Reply:
I agree, it would be better if we crete a python script or something like that, but it will take more time to do. When I have time I will do that